pkg/build/builder/cmd/scmauth/password.go

```diff
@@ -72,12 +72,12 @@ func (u UsernamePassword) Setup(baseDir string) error {
 
```

Comment:
So I guess we should defer step 3 above for the username, and only do it if we have a password. Otherwise, if no password and no username, return.